src/xcode/ENA/ENA/Source/Views/BaseElements/ENAButton.swift

```diff
@@ -102,6 +118,13 @@ class ENAButton: DynamicTypeButton {
 			setTitleColor(style.disabledForegroundColor.withAlphaComponent(0.5), for: .disabled)
 		}
 
+		if isLoading {
+			activityIndicator.color = style.disabledForegroundColor.withAlphaComponent(0.5)
```

Comment:
Please check if enabled and set either `foregroundColor` or `backgroundColor`. Otherwise this will result in wrong appearance:
```
isLoading = true
isEnabled = true
```